payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AddressElement.kt

```diff
@@ -36,6 +34,8 @@ open class AddressElement constructor(
     sameAsShippingElement: SameAsShippingElement?,
     shippingValuesMap: Map<IdentifierSpec, String?>?
 ) : SectionMultiFieldElement(_identifier) {
+    @VisibleForTesting
+    internal var isPlacesAvailable: IsPlacesAvailable = DefaultIsPlacesAvailable()
```

Comment:
Can we move this to the AddressElement constructor so that we don't need to make it visible for testing?

Reply:
Definitely. Good call.